crates/superposition/src/organisation/types.rs

```diff
@@ -0,0 +1,55 @@
+use serde::{Deserialize, Serialize};
+use superposition_types::database::models::organisation::OrgStatus;
+use superposition_types::database::models::organisation::Organisation;
+
+// Request payload for creating an organisation
+#[derive(Deserialize)]
+pub struct CreateOrganisationRequest {
+    pub country_code: Option<String>,
+    pub contact_email: Option<String>,
+    pub contact_phone: Option<String>,
+    pub created_by: String,
```

Comment:
this should not be a request param, this should be taken from the User's username